lib/Http/RequestHelper.php

```diff
@@ -166,16 +166,8 @@ private function detectFrontendRequest(Request $request): bool
             return false;
         }
 
-        $excludePatterns = [
-            "/^\/admin.*/",
-            "/^\/install.*/",
-            "/^\/plugin.*/",
-        ];
-
-        foreach ($excludePatterns as $pattern) {
-            if (preg_match($pattern, $request->getRequestUri())) {
-                return false;
-            }
+        if (preg_match("/^\/admin.*/", $request->getRequestUri())) {
```

Comment:
I think we should use another delimiter here, e.g.
```suggestion
        if (preg_match("#^/admin.*#", $request->getRequestUri())) {
```

Reply:
Changed it to `@` because `isFrontendRequestByAdmin` is also using some regex with `@` as a delimiter, so we are consistend in the same Class